example/src/ModifyQuantity/ModifyQuantityPage.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { StyleSheet } from 'react-native';
+import { ModifyQuantity, Screen, useTheme } from 'smartway-react-native-ui';
+
+export const ModifyQuantityPage = () => {
+    const theme = useTheme();
+
+    const styles = StyleSheet.create({
+        container: {
+            alignItems: 'center',
+            marginTop: theme.sw.spacing.m,
+        },
+    });
+
+    const [value, setValue] = useState<string>('0');
+
+    const onAdd = () => {
+        setValue((parseInt(value) + 1).toString());
+    };
+    const onMinus = () => {
+        setValue((parseInt(value) - 1).toString());
+    };
+
+    return (
+        <Screen style={styles.container}>
+            <ModifyQuantity onMinus={onMinus} onAdd={onAdd} text={'Stick'} inputValue={value} />
+            <ModifyQuantity
+                disabled
```

Comment:
I think the API should not expose the disabled props, but a `minValue` and `maxValue` that controls when "plus" or "minus" actions should be disabled.

```tsx
const [value, setValue] = useState<number>(0);
<ModifyQuantity
    minValue={0}
    maxValue={100}
    value={value}
    onChange={value => setValue(value)}
    text={'Stick'}
    icon={'minus-fill'} />



interface Props {
    text: string;
    onAdd: () => void;
    disabled?: boolean;
    onMinus: () => void;
    style?: ViewStyle;
    inputValue: string;
}
export const ModifyQuantity = () => {
   const minusDisabled = value <= minValue;
    // ...
                <Pressable disabled={minusDisabled } hitSlop={8} onPress={onChange(value -1)}>
                    <Icon
                        size={28}
                        color={minusDisabled  ? '#919EAB3D' : theme.sw.colors.neutral[800]}
                        name={icon}
                    />
                </Pressable>

    // And same for plus
}  
```

Reply:
updated!